src/components/structures/MatrixChat.js

```diff
@@ -185,6 +185,7 @@ module.exports = React.createClass({
             enableGuest: this.props.enableGuest,
             guestHsUrl: this.getCurrentHsUrl(),
             guestIsUrl: this.getCurrentIsUrl(),
+            defaultDisplayName: this.props.config.default_device_name,
```

Comment:
This could do with being in propTypes (especially as it would be good to doc what happens when it's undefined, although presumably this is up to the Home Server?). Also it should probably be in camelCase to be consistent with the other props.